frontend/src/screens/peers/Peers.tsx

```diff
@@ -18,20 +23,19 @@ import {
   TableHeader,
   TableRow,
 } from "src/components/ui/table.tsx";
-import { toast } from "src/components/ui/use-toast.ts";
+import { toast } from "src/components/ui/use-toast";
 import { useChannels } from "src/hooks/useChannels";
 import { usePeers } from "src/hooks/usePeers.ts";
 import { useSyncWallet } from "src/hooks/useSyncWallet.ts";
 import { Node } from "src/types";
 import { request } from "src/utils/request";
-import { useCSRF } from "../../hooks/useCSRF.ts";
 
 export default function Peers() {
   useSyncWallet();
-  const { data: peers, mutate: reloadPeers } = usePeers();
+  const { data: peers } = usePeers();
   const { data: channels } = useChannels();
   const [nodes, setNodes] = React.useState<Node[]>([]);
-  const { data: csrf } = useCSRF();
+  const [openPeerId, setOpenPeerId] = React.useState<string | null>(null);
```

Comment:
what is an `open peer`

Reply:
why do we need multiple dialogs? can't you just render the dialog component outside of the table?

Reply:
I thought the Trigger is necessary for the Dialog to function, fixed now, should be good. I think this way we can also simplify the transactions list. Wil also have a look there